### ISSUE TITLE
OPCToolbox_VS2017.sln - LINK : fatal error LNK1104: cannot open file 'OTBD.lib'

### ISSUE THREAD

**Issue by WaldemarCoding** [SoftingIndustrial/OPC-Classic-SDK#3]
When I try to compile the OPCToolbox_VS2017.sln I get a Link Error for OTBD.lib file.
I followed the _5._Compiling_the_OPC_Toolkit_core.md_ doc 
- uncommented OTBcfg.h
- commented license hint in OTB.cpp->OTActivate, but didn't change the default license key value (it's still "!! ADD YOUR LICENSE CODE HERE !!")
- set OTBFunctions.cs to use Debug version
- set the PlatformToolset to v142

I ran VS2019 16.11.2 in UserMode and AdminMode with the same result

```
1>Generating Code...
1>   Creating library .\bin\VS2017\Win32\Debug\OTBD.lib and object .\bin\VS2017\Win32\Debug\OTBD.exp
1>OTB_VS2017.vcxproj -> C:\Users\PC183\Desktop\OPC-Classic-SDK\development\core\src\OTB\bin\VS2017\Win32\Debug\OTBD.dll
1>        1 Datei(en) kopiert.
1>        1 Datei(en) kopiert.
1>Done building project "OTB_VS2017.vcxproj".
2>------ Build started: Project: OPCToolboxC++_VS2017, Configuration: Debug Win32 ------
2>LINK : fatal error LNK1104: cannot open file 'OTBD.lib'
2>Done building project "OPCToolboxC++_VS2017.vcxproj" -- FAILED.
========== Build: 1 succeeded, 1 failed, 1 up-to-date, 0 skipped ==========
```

**Comment by mrsuciu** [comment]
The VS2019 version seems to have changed somehow the behavior of the PostBuild copy event.
Try the following:

1. Compile the OTB_VS2017 project
2. Manually  create a folder called "lib" under "\development\core" and copy the "\development\core\src\OTB\bin\VS2017\Win32\Debug\OTBD.lib" to it.
3. Compile the OPCToolboxC++_VS2017 project

**Comment by FischerSeb** [comment]
@WaldemarCoding If you can solve the issue please feel free to make a PR

**Comment by WaldemarCoding** [comment]
Manually copying the file works. I also was able to solve it by switching to xcopy. Source: [StackOverflow](https://stackoverflow.com/a/9892956/6618411)

The PostBuild of OTB now looks like this
```
xcopy "$(TargetPath)" "..\..\..\bin\" /F /R /Y /I
xcopy "$(TargetDir)$(TargetName).lib" "..\..\lib\" /F /R /Y /I
```
instead of
```
copy "$(TargetPath)" ..\..\..\bin
copy "$(TargetDir)\$(TargetName).lib" ..\..\lib
```
Also notice that I removed the extra backslash between TargetDir and TargetName, as TargetName already contains the backslash and added quotes around the TargetDir.

@FischerSeb 
For the PR I would add VS2019 Versions of Projects and Solution:
- OPCToolboxC++_VS2019.vcxproj
- OTB_VS2019.vcxproj
- OPCToolboxNET_VS2019.csproj
- OPCToolbox_VS2019.sln
- .gitignore File (VisualStudio Setting from gitignore.io -> [link](https://www.toptal.com/developers/gitignore/api/visualstudio))
- the build also generates this 5 files (.bin, .h, .rc), should I add them to .gitignore or just don't stage them into the PR Commit?
![image](https://user-images.githubusercontent.com/17712059/133392759-0c9bad71-c941-4777-9ce9-6781530f4791.png)
